include/lsst/afw/geom/SpanSet.h

```diff
@@ -344,99 +344,123 @@ class SpanSet : public afw::table::io::PersistableFacade<lsst::afw::geom::SpanSe
      */
     std::shared_ptr<SpanSet> erode(SpanSet const & other) const;
 
-    /** @brief Create 1d array at points given by SpanSet
+    /** @brief Reduce the pixel dimensionality from 2 to 1 of an array at points given by SpanSet
      *
-     * Take values from an array at points defined by SpanSet and use them to populate a new 1D ndarray
+     * Take values from an array at points defined by SpanSet and use them to populate a new array
+     * where the x,y coordinates of the SpanSet have been flattened to one dimension. First two
+     * dimensions of the input array must be the h,w which correspond to the SpanSet coordinates. Any
+     * number of remaining dimensions is permissable.
      *
      * @param input - The ndarray from which the values will be taken
      * @param xy0 - A point object with is used as the origin point for the SpanSet coordinate system
```

Comment:
I don't believe we should be using dashes to separate terms in Doxygen `@param` calls - I think that they'll actually be rendered as dashes in the HTML, which looks weird. If you can confirm that's not the case, it's okay to leave them.